modules/twinklewelcome.js

```diff
@@ -323,10 +323,10 @@ Twinkle.welcome.templates = {
 				linkedArticle: true,
 				syntax: '{{subst:welcomenpov|$ARTICLE$|$USERNAME$}} ~~~~'
 			},
-			welcomeunsourced: {
+			'welcome-unsourced': {
```

Comment:
Can we alphabetize this please? We can do this by cutting and pasting this block of code a little higher.